python/test/test_polars.py

```diff
@@ -477,6 +478,25 @@ def test_polars_threshold():
         },
     )
 
+    actual = (
+        context.query()
+        .group_by("A")
+        .agg(pl.len().dp.noise())
+        .accuracy()  # type: ignore[union-attr]
+    )
+
+    expected = pl.DataFrame({
+        "column": ["len"],
+        "aggregate": ["Len"],
+        "distribution": ["Integer Laplace"],
+        "scale": [2.0]
+    })
+
+    # check that no threshold is set when keys are known
+    pl_testing.assert_frame_equal(actual, expected)
+
+    # check that query runs.
+    # output should be two columns ("A" and "len") with one row (1, ~1000)
```

Comment:
Thanks for adding this. Not something to block this PR, but how would you feel about including this in another `print` statement? Particularly if a test has multiple outputs, could make it easier to align the comment with the appropriate output.